mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PlaceAutocompletePresenter.java

```diff
@@ -1,22 +1,34 @@
 package com.mapzen.places.api.internal;
 
+import com.mapzen.android.lost.api.LocationServices;
+import com.mapzen.android.lost.api.LostApiClient;
 import com.mapzen.pelias.BoundingBox;
 import com.mapzen.pelias.gson.Properties;
 import com.mapzen.pelias.gson.Result;
 import com.mapzen.places.api.AutocompleteFilter;
 import com.mapzen.places.api.LatLngBounds;
 
+import android.location.Location;
+import android.util.Log;
+
 import retrofit2.Response;
 
 /**
  * Place autocomplete presenter to handle non-Android logic.
  */
-class PlaceAutocompletePresenter {
+class PlaceAutocompletePresenter implements LostApiClient.ConnectionCallbacks {
+  private static final String TAG = "MapzenPlaces";
+  private static final double BOUNDS_RADIUS = 0.02;
+  private static final double LAT_DEFAULT = 0.0;
+  private static final double LON_DEFAULT = 0.0;
+
   private final PlaceDetailFetcher detailFetcher;
   private final OnPlaceDetailsFetchedListener detailFetchListener;
   private final FilterMapper filterMapper;
   private LatLngBounds bounds;
   private AutocompleteFilter filter;
+  private LostApiClient client;
+  private boolean connected = false;
```

Comment:
Do we need a separate flag to track connection state or can we replace with `LostApiClient.isConnected()`?